### DISCUSSION TITLE
Thank you for this wonderful GUI crate!

### DISCUSSION THREAD

**Discussion by cessen** [emilk/egui#860] [General]
I'm currently working on building various VFX tools for use at my job.  I've always dreaded writing GUI code, but using egui I'm actually *enjoying* writing GUI code for the first time.  Moreover, it's been trivial to compile on both Linux and Windows (we use both at my work).

This is really an amazing crate, and I'm super grateful for all the time you've put into this.  Thank you!

**Comment by emilk** [comment]
Thanks 😊 ❤️
